server/chat/backend/agent/tools/mcp_tools.py

```
@@ -234,16 +232,9 @@ async def start_mcp_server(self, server_type: str, user_credentials: Dict = None
-                # Build Docker command with token passed via -e flag
-                # Using GITHUB_TOOLSETS=all to enable all 60+ tools
-                cmd = [
-                    docker_cmd, "run", "-i", "--rm",
-                    "-e", f"GITHUB_PERSONAL_ACCESS_TOKEN={github_token}",
-                    "-e", "GITHUB_TOOLSETS=all",
-                    "ghcr.io/github/github-mcp-server"
-                ]
+                cmd = [npx_cmd, "-y", "@modelcontextprotocol/server-github"]
                 
-                logging.info(f" GitHub MCP server command prepared (Docker with all toolsets)")
+                logging.info(f" GitHub MCP server command prepared (npx with all toolsets)")
```

Ruff (0.15.11)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)